### DISCUSSION TITLE
mateffy/pan-analytics-viewer

### DISCUSSION THREAD

**Discussion by Capevace** [HassanZahirnia/laravel-package-ocean#239] [Package Suggestions]
Hey there!

I just published the [`mateffy/pan-analytics-viewer`](https://github.com/Capevace/pan-analytics-viewer) package that shows a neat little UI for Nuno Maduro's [Pan](https://github.com/panphp/pan) analytics directly in the app's own UI. You see the analytics for the button you're hovering over.

Maybe it's useful enough to be included on the site?

Thanks for taking a look anyway!

<br>

![screenshot](https://github.com/user-attachments/assets/1e4a5336-cc9e-4fe5-b74a-5b6955a34f55)

https://github.com/user-attachments/assets/458618ce-ad5e-47bc-a0ad-8df1fe41056b

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
